CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ See [Porting Guide](http://docs.ansible.com/ansible/devel/porting_guides/porting
 * Removed restriction from protocol in cloudflare_dns module to allow other protocols than tcp and udp to be specified.
 
 * Ansible 2.6 and onwards, `target_id` parameter in `vmware_target_canonical_facts` module is an optional parameter.
+* `postgresql_user` module changed `encrypted=yes` to be the default. This
```

Comment:
I am extremely sorry, but Ansible procedure has changed since I suggested, you need to add changelog [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments).

Reply:
@Akasurde not a problem. I added a fragment. Just out of curiosity where can I read the docs about these changelog fragments, because if there was no fragments, I wouldn't know how to write it.